common/Android.mk

```diff
@@ -131,6 +131,8 @@ LOCAL_CPPFLAGS += \
         -Wcast-qual -Wcast-align \
 	-DLOCK_DIR_PREFIX='"/vendor/etc"' \
         -DHWC_DISPLAY_INI_PATH='"/vendor/etc/hwc_display.ini"' \
+        -DKVM_HWC_DISPLAY_INI_PATH='"/vendor/etc/hwc_display.kvm.ini"' \
```

Comment:
What about still using original hwc_display.ini, after parsing it, then check if KVM platform to adjust the available planes? This will introduce less modification, no need an extra config file. I suppose you are trying to make it general for specific platform/use case, but currently there is no other config requests than plane limitation right?

Reply:
Yes, I think so too. but After discussing with Di, She is worrying if we need to hard code the plane limitation for KVM. On QEMU, hwc can not use the real hardware planes in current implementation, before committing to the real hardware plane, QEMU need to compose all planes, so She think it will be better to render on only one plane. but in the future version, it is possible that hwc can commit to the real hardware plane directly (maybe share with Linux Host), So she suggest to add a new config for KVM.